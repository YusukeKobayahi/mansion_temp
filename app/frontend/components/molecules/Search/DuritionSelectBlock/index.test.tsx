import { render } from "@testing-library/react";
import DuritoinSelectBlock from ".";

describe("components/pages/commute/stations/Search/DuritionSelectBlock", () => {
  it("正常に表示される", () => {
    // 本来SelectBoxをchildrenとして渡すためのコンポーネントだが、テストでは、SelectBoxの挙動に依存しないためにSelectBoxを渡していない。
    const { asFragment } = render(
      <DuritoinSelectBlock
        labelText={"平均相場"}
        minSelectBox={<button>テスト</button>}
        maxSelectBox={<button>テスト</button>}
      />
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
