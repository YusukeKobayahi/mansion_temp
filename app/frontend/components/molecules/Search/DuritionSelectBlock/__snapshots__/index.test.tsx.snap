// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/pages/commute/stations/Search/DuritionSelectBlock 正常に表示される 1`] = `
<DocumentFragment>
  <div
    class="block sectionBar"
  >
    <div
      class="label"
    >
      <p>
        平均相場
      </p>
    </div>
    <div
      class="inputs"
    >
      <button>
        テスト
      </button>
      <span>
         ~ 
      </span>
      <button>
        テスト
      </button>
    </div>
  </div>
</DocumentFragment>
`;
